src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -152,6 +158,22 @@
       <Name>System.Globalization</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.7'">
```

Comment:
this will require moving the whole tests and not only the tests I am adding. I would say if we decide to do that we can do it in different PR. but IMHO I would say keeping the tests here is logical too as it is testing old contracts too in addition to the new one.